ocf_datapipes/load/satellite.py

```diff
@@ -164,22 +192,46 @@ def open_sat_data(zarr_path: Union[Path, str, list[Path], list[str]]) -> xr.Data
     return data_array
 
 
+def check_last_timestamp(dataset: xr.Dataset, timedelta_hours:float= 1) -> bool:
+    """
+    Check the last timestamp of the dataset to see if it is more than 1 hour ago
+
+    Args:
+        dataset: dataset with time dimension
+        timedelta_hours: the timedelta to check from now
+
+    Returns: bool
+    """
+    latest_time = pd.to_datetime(dataset.time[-1].values)
+    if latest_time < datetime.now(tz=timezone.utc) - timedelta(hours=timedelta_hours):
```

Comment:
@peterdudfield does this work with an explicit UTC timezone in `datetime.now(tz=timezone.utc)`? I've had errors before related to comparing datetimes with explicit timezones to those without explicit timezones. I think the `latest_time`  shouldn't have an explicit timezone, but I may be wrong

Reply:
you are right, I do remember that error. I'm hopping I added some unittest tests, let me look